doc/logos/README.md

```diff
@@ -1,52 +1,63 @@
-# scikit-learn Branding
+# scikit-learn Brand Guidelines
+
+This section contains information around scikit-learn's brand standards and the use of scikit-learn assets. The purpose of these guidelines are to ensure the scikit-learn logo remains consistent and recognizable across all uses and communications. These guidelines also provide a common language for referring to the logos and their components.
+
 
-This section contains scikit-learn's branding standards.
 File types:
 - `PNG` is a higher-quality compression format; size of file is generally larger
 - `ICO` file format refers to an image file format that contains small size computer icon images
 - `SVG` Scalable Vector Graphics (SVG) are an XML-based markup language for describing two-dimensional based vector graphics. They can be created and edited with any text editor or with drawing software.
 
 ---
+<br>
 
-# scikit-learn: Font, Colors & Logos
 
-## scikit-learn Font
-- "scikit": Helvetica Neue
-- "learn": Script MT
+## Brand Name
+The official name of the package is __scikit-learn__. Do not abbreviate or otherwise alter the name. Always spell ‘scikit’ with a lowercase ‘s’.
+
```

Comment:
+1000%

(since people often mistakenly call it sci-kit and it drives me crazy)